delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/Realms.scala

```diff
@@ -0,0 +1,74 @@
+package ch.epfl.bluebrain.nexus.delta.sdk
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.delta.sdk.model.realms.RealmRejection
+import ch.epfl.bluebrain.nexus.delta.sdk.model.realms.RealmRejection.RevisionNotFound
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{Label, Name, SearchParams}
+import monix.bio.{IO, Task}
+
+/**
+  * Operations pertaining to managing realms.
+  */
+trait Realms {
+
+  /**
+    * Creates a new realm using the provided configuration.
+    *
+    * @param label        the realm label
+    * @param name         the name of the realm
+    * @param openIdConfig the address of the openid configuration
+    * @param logo         an optional realm logo
+    */
+  def create(label: Label, name: Name, openIdConfig: Uri, logo: Option[Uri]): IO[RealmRejection, RealmResource]
+
+  /**
+    * Updates an existing realm using the provided configuration.
+    *
+    * @param label        the realm label
+    * @param rev          the current revision of the realm
+    * @param name         the new name for the realm
+    * @param openIdConfig the new openid configuration address
+    * @param logo         an optional new logo
+    */
+  def update(
```

Comment:
Should we put the rev as the last parameter?

Reply:
the general pattern that we're using is: id, rev and then the others. I think it's fine to keep it as is.